ioread.cpp

```diff
@@ -790,14 +793,17 @@ template<unsigned long int N> bool readFsGridVariable(
    attribs.push_back(make_pair("name",variableName));
    attribs.push_back(make_pair("mesh","fsgrid"));
 
+   phiprof::start("getArrayInfo");
    if (file.getArrayInfo("VARIABLE",attribs,arraySize,vectorSize,dataType,byteSize) == false) {
       logFile << "(RESTART)  ERROR: Failed to read " << endl << write;
       return false;
```

Comment:
This return fails to close the phiprof group just opened above.
(I suppose we are anyway going to fail in this case, but it would be good style to always have closed groups, also in crashing cases)